src/App.tsx

```diff
@@ -1,26 +1,151 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useRef, useState } from 'react';
+import { FilterType } from './types/FilterType';
+import { ErrorMessages } from './types/ErrorMessages';
+import { Todo } from './types/Todo';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import * as todoSetvices from './api/todos';
+import { Footer } from './components/Footer';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessages>(
+    ErrorMessages.default,
+  );
+  const [currentFilter, setCurrentFilter] = useState(FilterType.All);
+  const [isLoading, setIsLoading] = useState(false);
+  const [loadingIds, setLoadingIds] = useState<number[]>([]);
+
+  const [inputValue, setInputValue] = useState('');
+
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  const inputRef = useRef<HTMLInputElement | null>(null);
+
+  useEffect(() => {
+    if (!isLoading && inputRef.current) {
+      inputRef.current.focus();
+    }
+  }, [isLoading]);
+
+  const getTodoFromServer = () => {
+    setIsLoading(true);
+
+    todoSetvices
+      .getTodos()
+      .then(setTodoList)
+      .catch(() => {
+        setErrorMessage(ErrorMessages.getError);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  };
+
+  const deleteTodos = (todoId: number) => {
```

Comment:
```suggestion
  const deleteTodo = (todoId: number) => {
```
It works with one todo